recipes/flatcc/all/conanfile.py

```diff
@@ -99,12 +109,36 @@ def package(self):
                       os.path.join(self.package_folder, "bin", "flatcc"))
         # Copy license file
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        # Remove cmake config files
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", "flatcc"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", "flatccruntime"))
+        os.remove(os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "flatcccli-config.cmake"))
+        os.remove(os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "flatcccli-config-version.cmake"))
+        os.remove(os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "flatcccli-targets.cmake"))
+        os.remove(os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "flatcccli-targets-release.cmake"))
+        #Patch FlatccGenerateSources.cmake file until Conan supports the flatcc:cli executable target used in FlatccGenerateSources.cmake
+        genSourcesMod = os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "FlatccGenerateSources.cmake")
+        tools.replace_in_file(genSourcesMod, "flatcc::cli", "$ENV{FLATCC_CLI_EXE}", strict=True)
 
     def package_info(self):
-        bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info('Appending PATH environment variable: %s' % bin_path)
-        self.env_info.PATH.append(bin_path)
         debug_suffix = "_d" if self.settings.build_type == "Debug" else ""
+        #flatcc package provides two components: the flatcc compiler binary and the runtime library
         if not self.options.runtime_lib_only:
-            self.cpp_info.libs.append("flatcc%s" % debug_suffix)
-        self.cpp_info.libs.append("flatccrt%s" % debug_suffix)
+            self.cpp_info.components["cli"].names["cmake_find_package"] = "cli"
```

Comment:
Maybe add a FIXME about missing `flatcccli` and `flatccruntime` (2 files) instead of one
Conan is missing this feature.
Also FIXME about missing `flatcc::cli` target.

Reply:
> Maybe add a FIXME about missing `flatcccli` and `flatccruntime` (2 files) instead of one

Can you elaborate a bit? What exactly is missing ?

> Also FIXME about missing `flatcc::cli` target

Ok, I'll add a comment that `flatcc::cli` executable target currently doesn't work with Conan.

Reply:
Maybe it's unnecessary, but right now it's installing 3 cmake packages:
`flatcc-config.cmake`, `flatcccli-config.cmake` and flatccruntime-config.cmake`.
Of these, currently only `flatcc-config.cmake` is re-created by conan.

Reply:
But aren't flatcccli-config.cmake and flatccruntime-config.cmake more like 'internal' config files?
End-users should use find_package(flatcc).

Reply:
I've documented flatcccli for cross building purposes.
But anyhow, maybe add a FIXME/TODO for when conan receives support for creating multiple cmake modules.

Reply:
> I've documented flatcccli for cross building purposes.

I noticed. Further down in the README there is also a section called 'Cross-compilation' that I am updating now. Will create a pull request to your branch when done tomorrow.

> But anyhow, maybe add a FIXME/TODO for when conan receives support for creating multiple cmake modules.

Will do.

Reply:
> I've documented flatcccli for cross building purposes.

I noticed. Further down in the README there is also a section called 'Cross-compilation' that I am updating now. Will create a pull request to your branch when done tomorrow.

> But anyhow, maybe add a FIXME/TODO for when conan receives support for creating multiple cmake modules.

Will do.